### DISCUSSION TITLE
hide mode doesn't hide the i3 bar

### DISCUSSION THREAD

**Discussion by Saithe6** [i3/i3#6095] [Q&A] [answered]
In fact it seems to just summon a new i3 bar when I have my mod key held down. I have been trying to fix this for an hour now and I have no clue what might be causing it

**Comment by okraits** [comment]
Can you please share your config so we can help you?

**Comment by Saithe6** [comment, marked as answer]
Alright the answer was just I am stupid. At the very bottom of the autogenerated config file is a bar block. I promise I did search for a bar block within the config, I just missed this one. Sorry, but at least this is here for people in the future
